tmt/export/polarion.py

```diff
@@ -56,24 +56,26 @@ def get_polarion_ids(
 
     if not query_result:
         return 'None', None
-    if len(query_result) == 1:
+    if len(query_result) == 1 and query_result[0].status != 'inactive':
         return query_result[0].work_item_id, query_result[0].project_id
 
     if preferred_project:
         try:
             return next(
-                item.work_item_id for item in query_result if item.project_id == preferred_project
+                item.work_item_id
+                for item in query_result
+                if item.project_id == preferred_project and item.status != 'inactive'
             ), preferred_project
         except StopIteration:
             pass
 
     for result in query_result:
         # If multiple cases are found prefer cases from other projects
         # than these legacy ones
-        if str(result.project_id) not in LEGACY_POLARION_PROJECTS:
+        if str(result.project_id) not in LEGACY_POLARION_PROJECTS and result.status != 'inactive':
             return result.work_item_id, result.project_id
 
-    return query_result[0].work_item_id, query_result[0].project_id
+    return 'None', None
```

Comment:
Does `'None'`, a string `'None'`, have any special meaning for Polarion or the import? If it's used here just to signal "no case ID", then I'd strongly recommend making it an actual `None`, plus changing the return type to `Optional[str]` instead of `str`. Encoding "nothingness" into strings makes it terribly hard for linters to follow, it's easy to use `'None'` where a meaningful string is expected - `None` would be reported by a type checker, `'None'` wouldn't.